Describe workaround for recent Docker Desktop mishap (#27354)
mistercrunch/Airflow: BREEZE.rst

             alt="Disk space MacOS">
    </div>

- **Docker is not running** - even if it is running with Docker Desktop. The version of
  Docker Desktop released late October 2022 (4.13.0) has ``/var/run/docker.sock`` removed.
  If you install 4.13.0 for the first time you will miss ``/var/run/docker.sock`` and you will get
  "docker is not running" error. This was done too hastily and they are likely to

>   `remove it <https://github.com/docker/for-mac/issues/6529#issuecomment-1292135881i>`_ in the

Reviewer: ```suggestion
  `add it back <https://github.com/docker/for-mac/issues/6529#issuecomment-1292135881i>`_ in the
```

>   next patchlevel, but if you happen to see "docker is not running" when it is, you should not have

Reviewer: ```suggestion
  next patchlevel. If you happen to see "docker is not running" when it is, you may not have
```
